src/language/plain-text.md

```diff
@@ -37,3 +53,78 @@ script.
   else
     console.log('use pug')
 ```
+
+You can also create a dot block of plain text *after* other tags within the parent tag.
+
+```pug-preview
+div
+  p This text belongs to the paragraph tag.
+  br
+  .
+    This text belongs to the div tag.
+```
+
+## Whitespace Control
+
+Managing the whitespace of the rendered HTML is the trickiest part about learning Pug. By default, Pug removes spaces from the beginning and end of the lines for both inline-level tags and plain text lines and blocks. The value here is that it gives you full control over whether inline tags and/or plain text should touch. It even lets you place tags in the middle of words.
+
+```pug-preview
+| You put the em
+em pha
+| sis on the wrong syl
+em la
+| ble.
+```
+
+The trade-off is that it *requires* you to think about and take control over whether tags and text touch. Don't worry, though, you'll get the hang of it soon enough.
+
+If you need the text and/or inline-level tags to touch --- perhaps you need a period to appear outside the hyperlink at the end of a sentence --- this is easy, as it's basically what happens unless you tell Pug otherwise.
+
+```pug-preview
+a ...sentence ending with a link
+| .
+```
+
+If you need to *add* space, you have a few options:
+
+Depending on where you need the whitespace, you could add an extra space at the beginning of the plain text line (after the pipe for piped text). Or you could add a trailing space at the *end* of the plain text line. Less often, you might need extra space *inside the tag*, but it's the same idea --- extra spaces at the beginning of the text (after the tag), or at the end of the line. This may require configuring your code editor *not* to strip trailing spaces when you save, and you will need to make sure all contributors to your project do the same.
+
+**NOTE** the trailing and leading spaces here:
```

Comment:
I could definitely try to *discourage* the use of this workaround more, but I don't think it should be removed entirely.

This is indeed sort of a dangerous workaround for users or teams that have not all configured their editors properly. However: 

1. It does work. Users who discover this feature (or side-effect? I wrote about this aspect of Whitespace Control assuming it was a deliberate feature...) may not correctly intuit the rules about leading and trailing whitespace if it's not documented, leading to more annoying "bug" reports.
2. On the other hand, not *all* code editors automatically strip trailing whitespace, and some editor plugins for Pug *prevent* the usual whitespace stripping. In most languages, trailing whitespace is an aesthetic problem only, so it's possible a Pug user might leave trailing whitespace out of negligence and then be surprised by the results.